pkg/common/annotations.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+const (
+	// AnnotationName is the annotation on child resources that specifies which ArgoCD instance
+	// name a specific object is associated with
+	AnnotationName = "argocds.argoproj.io/name"
```

Comment:
Using the plural form of argocd (argocds) seems odd to me.   Is it intentional?

Reply:
The CRD is already named that way :) 

```
apiVersion: apiextensions.k8s.io/v1beta1
kind: CustomResourceDefinition
metadata:
  name: argocds.argoproj.io
spec:
  group: argoproj.io
  names:
    kind: ArgoCD
    listKind: ArgoCDList
    plural: argocds
```